documentation-site/components/theming/common.js

```diff
@@ -13,52 +13,116 @@ LICENSE file in the root directory of this source tree.
 
 import * as React from 'react';
 import {useStyletron, styled} from 'baseui';
+import {LightTheme, DarkTheme} from 'baseui/themes';
 
 const monospaceFontFamily =
   'SFMono-Regular, Consolas, "Liberation Mono", Menlo, Courier, monospace';
 
+const SubTitle = styled<{}>('span', ({$theme}) => {
+  return {
+    ...$theme.typography.LabelXSmall,
+    fontFamily: monospaceFontFamily,
+    fontStyle: 'italic',
+    color: $theme.colors.contentSecondary,
+  };
+});
+
 export const Title = styled<{}>('div', ({$theme}) => {
   return {
-    ...$theme.typography.HeadingXSmall,
+    ...$theme.typography.ParagraphLarge,
```

Comment:
Since it's a title it makes sense to use one of the Heading components.

Reply:
changed it back to a heading